app/javascript/react/components/Modals/SessionDetailsModal/SessionDetailsModal.style.tsx

```diff
@@ -155,6 +155,10 @@ const CancelButtonX = styled.button`
   border: none;
   background-color: transparent;
   width: fit-content;
+
+  &:focus-visible {
```

Comment:
DO NOT TAKE MY WORK (I already have less things) 😜

Reply:
😂